src/components/SearchFilter/SearchFilter.tsx

```diff
@@ -0,0 +1,231 @@
+import React, { useState } from 'react'
+import {
+  Grid,
+  Checkbox,
+  Dropdown,
+  Button,
+  Label,
+  Fieldset,
+} from '@trussworks/react-uswds'
+import styles from './SearchFilter.module.scss'
+import { useSearchContext } from 'stores/searchContext'
+
+type PropTypes = {
+  labels: {
+    name: string
+  }[]
+}
+
+const SearchFilter = ({ labels }: PropTypes) => {
+  const { searchQuery, setSearchQuery } = useSearchContext()
+
+  // Manages the state of the checkboxes and dropdown in the filter
+  const [filterItems, setFilterItems] = useState({
+    application: searchQuery.includes('category:application') ? true : false,
+    news: searchQuery.includes('category:news') ? true : false,
+    documentation: searchQuery.includes('category:documentation')
+      ? true
+      : false,
+    // A label can exist in two forms: label:labelName or label:"label name"
+    // If we split searchQuery at label: and there is a " immediately after, we know the label is in quotes
+    // and we need to get all the characters between the quotes. After we have the label, we need to remove
+    // the quotes and set the dropdown state to the label
+    dropdown:
+      searchQuery.includes('label:') &&
+      searchQuery.split('label:')[1].charAt(0) === '"'
+        ? searchQuery.split('label:')[1].split('"')[1].split('"')[0]
+        : searchQuery.includes('label:') &&
+          searchQuery.split('label:')[1].split(' ')[0],
```

Comment:
Shoutout to Copilot. I knew what I needed to implement here, but also knew that writing the code would be a pain. The comment was written by me, and I let Copilot do the rest. I only had to make the slight modification to check `searchQuery.includes(label:)`.